sdk/datalake-analytics/Azure.ResourceManager.DataLakeAnalytics/api/Azure.ResourceManager.DataLakeAnalytics.netstandard2.0.cs

```diff
@@ -434,151 +392,192 @@ public enum DataLakeAnalyticsAccountStatus
+        public static bool operator !=(Azure.ResourceManager.DataLakeAnalytics.Models.DataLakeAnalyticsSubscriptionState left, Azure.ResourceManager.DataLakeAnalytics.Models.DataLakeAnalyticsSubscriptionState right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public enum TierType
+    public partial class DataLakeAnalyticsVirtualNetworkRule : Azure.ResourceManager.Models.ResourceData
     {
-        Consumption = 0,
-        Commitment100AUHours = 1,
-        Commitment500AUHours = 2,
-        Commitment1000AUHours = 3,
-        Commitment5000AUHours = 4,
-        Commitment10000AUHours = 5,
-        Commitment50000AUHours = 6,
-        Commitment100000AUHours = 7,
-        Commitment500000AUHours = 8,
+        internal DataLakeAnalyticsVirtualNetworkRule() { }
+        public Azure.Core.ResourceIdentifier SubnetId { get { throw null; } }
+        public Azure.ResourceManager.DataLakeAnalytics.Models.DataLakeAnalyticsVirtualNetworkRuleState? VirtualNetworkRuleState { get { throw null; } }
     }
-    public partial class UpdateComputePolicyWithAccountParameters
+    public enum DataLakeAnalyticsVirtualNetworkRuleState
     {
-        public UpdateComputePolicyWithAccountParameters(string name) { }
-        public int? MaxDegreeOfParallelismPerJob { get { throw null; } set { } }
-        public int? MinPriorityPerJob { get { throw null; } set { } }
-        public string Name { get { throw null; } }
-        public System.Guid? ObjectId { get { throw null; } set { } }
-        public Azure.ResourceManager.DataLakeAnalytics.Models.AADObjectType? ObjectType { get { throw null; } set { } }
+        Active = 0,
+        NetworkSourceDeleted = 1,
+        Failed = 2,
     }
-    public partial class UpdateDataLakeStoreWithAccountParameters
+    public partial class DataLakeStoreAccountInformationCreateOrUpdateContent
     {
-        public UpdateDataLakeStoreWithAccountParameters(string name) { }
-        public string Name { get { throw null; } }
+        public DataLakeStoreAccountInformationCreateOrUpdateContent() { }
         public string Suffix { get { throw null; } set { } }
     }
-    public partial class UpdateFirewallRuleWithAccountParameters
+    public partial class DataLakeStoreForDataLakeAnalyticsAccountCreateOrUpdate
     {
-        public UpdateFirewallRuleWithAccountParameters(string name) { }
-        public string EndIPAddress { get { throw null; } set { } }
+        public DataLakeStoreForDataLakeAnalyticsAccountCreateOrUpdate(string name) { }
         public string Name { get { throw null; } }
-        public string StartIPAddress { get { throw null; } set { } }
+        public string Suffix { get { throw null; } set { } }
     }
-    public partial class UpdateStorageAccountWithAccountParameters
+    public partial class DataLakeStoreForDataLakeAnalyticsAccountUpdate
```

Comment:
Shall we end with a noun like `DataLakeStoreForDataLakeAnalyticsAccountUpdateContent`?